.travis.yml

```diff
@@ -12,6 +12,7 @@ language: node_js
 node_js:
   - '8'
   - '10'
+  - '12'
```

Comment:
needs appveyor as well? or alternatively change the commit description to say we only support node 12 on linux/mac?

Reply:
also not sure why that's a ```feat```